src/parsing/processProlog.ts

```diff
@@ -88,6 +88,12 @@ export default function processProlog(
 
 		staticContext.registerNamespace(prefix, namespaceURI);
 	});
+	astHelper.getChildren(prolog, 'defaultNamespaceDecl').forEach(defaultNamespaceDecl => {
```

Comment:
There may only be 1 default namespace declaration.
From spec: "A Prolog may contain at most one default function namespace declaration [err:XQST0066]."